src/csv_to_json.py

```diff
@@ -0,0 +1,65 @@
+import csv
+import yaml
+import json
+from pathlib import Path
+
+class bcolors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKCYAN = '\033[96m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+def main():
+    input_dir = Path('csv')
+    output_dir = Path('json')
+    output_dir.mkdir(parents=True, exist_ok=True)
+    mapping_path = Path('src/config.yaml')
+
+    with open(mapping_path, 'r', encoding='utf-8') as f:
+        config = yaml.safe_load(f)
+
+    spreadsheet_entries = config['spreadsheet']
+    skip = []
+    for e in spreadsheet_entries:
+        if e['skip'] is True:
+            skip.append(e['name'])
+
+    mapping_list = config['mapping']
+    mapping = {item['tess']: item['schema'] for item in mapping_list}
+
+    for csv_file in input_dir.glob('*.csv'):
+        name = str(csv_file).removeprefix('csv/').removesuffix('.csv')
+        if name in skip:
+            continue
+
+        with open(csv_file, 'r', encoding='utf-8') as f:
+            reader = csv.DictReader(f)
+            mapped_fields = [k for k in mapping if k in reader.fieldnames]
+            materials = []
+
+            for row in reader:
+                resource = {
+                    '@context': 'https://schema.org',
```

Comment:
```suggestion
                    '@context': 'http://schema.org',
```
It is still recommended to go for `http`, even though it will move at some point to `https`